test/e2e/e2e_suite.go

```diff
@@ -0,0 +1,85 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
```

Comment:
I wonder if something like "e2e_helpers" or "e2e_utils" would be a more appropriate name for this file

Reply:
I would actually question why we need a new file at all. Between the existing `test/e2e/e2e_suite_test.go` and `test/e2e/helpers.go` do we not have the appropriate places for these additions?

Reply:
oh nevermind it's literally *the thing* that solves the stated problem

Reply:
I could move it to `test/e2e/helpers.go` I think.  the comments https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/fb0a7f425c238eeeb6d1bd3ed16254f0884fb8d8/test/e2e/e2e_suite.go#L16 made me think it should be here for the variables but the `validateStableReleaseString` should probably go to `helpers.go`. wdyt?

Reply:
> What about naming it `e2e_suite_vars.go` or something like that?

sounds good.

> Side note, does CAPI have the same issue? I've noticed they also have test suite flags and vars in the main `e2e_suite_test.go` file

Just looked. It seems they have a slightly different set up. Where the files that reference variables that these variables always use `_test.go` and they have another file that live along side it. For example:

https://github.com/kubernetes-sigs/cluster-api/blob/main/test/e2e/cluster_upgrade.go
https://github.com/kubernetes-sigs/cluster-api/blob/main/test/e2e/cluster_upgrade_test.go